lib/data-url.js

```diff
@@ -0,0 +1,17 @@
+"use strict"
+
+const dataURLRegexp = /^data:text\/css;base64,/i
+
+function isValid(url) {
+  return dataURLRegexp.test(url)
+}
+
+function contents(url) {
+  // "data:text/css;base64,".length == 21
```

Comment:
I lint comments too :stuck_out_tongue_winking_eye: 
```suggestion
  // "data:text/css;base64,".length === 21
```